dockers/docker-sonic-gnmi/gnmi-native.sh

```diff
@@ -71,9 +71,9 @@ else
 fi
 
 # Enable ZMQ for SmartSwitch
-LOCALHOST_SUBTYPE=`sonic-db-cli CONFIG_DB hget localhost "subtype"`
+LOCALHOST_SUBTYPE=`sonic-db-cli CONFIG_DB hget "DEVICE_METADATA|localhost" "subtype"`
 if [[ x"${LOCALHOST_SUBTYPE}" == x"SmartSwitch" ]]; then
```

Comment:
@prsunny  Do you know where is the HLD about subtype==SmartSwitch?

Reply:
@r12f

Reply:
DEVICE_METADATA|localhost: 
-	type: “LeafRouter”, 
-	subtype: “SmartSwitch”   
-	dpu: “Number of DPUs”

Reply:
@Pterosaur , its part of HLD